apps/admin-gui/src/app/vos/pages/vo-detail-page/vo-members/vo-members.component.ts

```diff
@@ -80,12 +76,10 @@ export class VoMembersComponent implements OnInit {
     this.route.parent.params.subscribe(parentParams => {
       const voId = parentParams['voId'];
 
-      this.isManualAddingBlocked(voId).then(() => {
-        this.voService.getVoById(voId).subscribe(vo => {
-          this.vo = vo;
-          this.setAuthRights();
-          this.loading = false;
-        });
+      this.voService.getVoById(voId).subscribe(vo => {
+        this.vo = vo;
+        this.setAuthRights();
+        this.loading = false;
```

Comment:
but we need to check if manual adding to VO is allowed
this is misuderstanding probably - if blockManualMemberAdding(VO attribute) to VO is true, then we need to forbid adding members to VO and GROUP from outside of VO
blockManualMemberAdding(group attribute) is true then we need to forbid adding members overall